tests/.deprc

```diff
@@ -1,3 +1,4 @@
   {
-    "sass_root": "scss/" 
+    "sass_root": "scss/",
+    "stylus_root": "stylus/"
```

Comment:
`styles_root` now. It's meant to be generic enough to support all preprocessors. `styles` is super close to `stylus`, I know :)